python/ray/actor.py

```diff
@@ -109,10 +109,16 @@ def annotate_method(method):
 # Create objects to wrap method invocations. This is done so that we can
 # invoke methods with actor.method.remote() instead of actor.method().
 class ActorMethod(object):
-    def __init__(self, actor, method_name, num_return_vals):
+    def __init__(self, actor, method_name, num_return_vals, decorator=None):
         self._actor = actor
         self._method_name = method_name
         self._num_return_vals = num_return_vals
+        # This is a decorator that is used to wrap the function invocation (as
```

Comment:
This should be in the docstring of the __init__ method, and also point the user to an example on how to use it.